pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -462,13 +458,9 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 				fmt.Sprintf("invalid entry metadata.annotations.%s", hooks.HookSidecarListAnnotationName),
 			),
 		)
-
-		DescribeTable("should accept annotations which require feature gate enabled", func(annotations map[string]string, featureGate string) {
+		DescribeTable("should accept annotations which require feature gate enabled", func(annotationKey, annotationValue, featureGate string) {
```

Comment:
Please consider keeping the annotations as a map, as it makes the test shorter and keeps the key-value pairs together.

Reply:
Must have missed this, fixing.